intl/en/messages.json

```diff
@@ -0,0 +1,39 @@
+{
+
+    "msgAll1": "usage: {{%MAIN%}} {{[}}options{{]}} {{[URL [PACK|BRANCH]]}}\n\nDeploy a node application to a StrongLoop process manager\n\nOptions:\n  {{-h,--help}}          Print this message and exit.\n  {{-v,--version}}       Print version and exit.\n  {{-s,--service SVC}}   Deploy to service {{SVC}}.\n  {{-z,--size SIZE}}     Set size of cluster to {{SIZE}} before deploy.\n\nArguments:\n  {{SIZE}}: A number, n, or the string {{'cpus'}}, meaning run a worker per CPU.\n\n  {{SVC}}: The name or ID of the service on the process manager.\n\n    Defaults to the name of the application specified in the {{package.json}} file.\n\n    If a service with that name is not available on the {{process manager}},\n    a new service will be created and the application will be deployed.\n\n  {{URL}}: The URL of the {{StrongLoop process manager}}.\n\n    Defaults to {{`http://localhost:8701`}}. If a URL is provided, both the host\n    and port is optional, they default to {{`localhost`}} and 8701, respectively.\n    If the server requires authentication, the credentials must be part of the\n    URL, see examples.\n\n    {{URL}} may also use the {{`http+ssh://`}} protocol which will connect using ssh\n    and then tunnel the http requests over that connection. The ssh username\n    will default to your current user and authentication defaults to using your\n    current ssh-agent. The username can be overridden by setting an {{`SSH_USER`}}\n    environment variable.",
```

Comment:
This is impossible to maintain. Encoding entire text files as a single multi-line string isn't going to work.

I think we will need an alternative to json, something like:

- `intl/en/sl-deploy.txt`: this will be translated into `intl/*/sl-deploy.txt` for the other languages